README.md

```diff
@@ -1,12 +1,74 @@
-# About
-GitHub Action to execute tests by Testing Farm and update Pull Request status.
+# testing-farm-as-github-action
 
-The Action uses ubuntu-20.04
+Testing Farm as GitHub Action is a GitHub Action to execute tests by Testing Farm and update Pull Request status.
 
-## Prerequisities
+This action only runs on `Linux`. Before a calling this GitHub Action, you must first clone your project by e.g. 
```

Comment:
```suggestion
This action internally runs on ubuntu-20.04. Before calling this GitHub Action, you must first clone your project by, e.g., 
```
I have checked the code, and it does not seem, that the user can choose the base Linux distribution for this action. I would say, that is sufficient to state, that the action runs on ubuntu.

Reply:
Interesting, good to know, thanks :).

Reply:
Fixed by https://github.com/sclorg/testing-farm-as-github-action/pull/6